onyo/lib/onyo.py

```diff
@@ -95,6 +95,7 @@ def __init__(self,
 
         # caches
         self._asset_paths: list[Path] | None = None
+        self._config_cache: dict[str, dict[str, str]] = dict()
```

Comment:
Why not here (and below in `set_config()`) initialize to:
```suggestion
        self._config_cache: dict[str, dict[str, str]] = {'onyo': {}, 'git':{}}
```

instead of calling `setdefault()` everytime (L198 and 205)?

Reply:
I agree. Part of my leftover desire to use `None`, that I abandoned.

I have fixed this as well, and will push soon once the tests pass locally.